src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,59 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Car car = new Car();
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturer = new Manufacturer(1L, "Nisan", "Japan");
+        car.setManufacturer(manufacturerService.create(manufacturer));
+        car.setModel("X1");
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        List<Driver> drivers = new ArrayList<>();
+        Driver driverBob = new Driver(1L, "Bob", "qw5890");
```

Comment:
You should not set the ID
The DB creates IDs `AUTO_INCREMENT`
```suggestion
        Driver driverBob = new Driver("Bob", "qw5890");
```